src/components/Clusters/views/ClusterOverview/ClusterStats.js

```diff
@@ -72,6 +95,71 @@ export default function ClusterStats({ data }) {
             }}
           />
         </Card>
+        <CardWithTooltip
```

Comment:
We need to not display this section or display some info if `podsData` and `deploymentsData` is not available, because for now it's empty card with title only 
![image](https://github.com/kyma-project/busola/assets/46759954/b12224e6-c893-4cc8-89df-c8ee15972bec)